src/main/java/com/hms/flexyosisoftconnector/configuration/OSIsoftConfig.java

```diff
@@ -394,15 +346,6 @@ public static String getCertificatePath() {
     return ewonCertificatePath;
   }
 
-  /**
-   * Get the target URL
-   *
-   * @return the target URL
-   */
-  public static String getTargetURL() {

```

Comment:
keep this function (unless it is not used anywhere). return piServerUrl; or return GetServerUrl()